test/other/codesize/test_codesize_minimal_pthreads.gzsize

```diff
@@ -1 +1 @@
-4037
+4045
```

Comment:
Code size increase seems fine as this is just for node.js (and pthreads) logic.

Reply:
Indeed.